src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
+$em-base: $base-font-size
+
+// Colors
+$c-main: green
```

Comment:
Umm, tomuhle vůbec nerozumím. Úplně každý web má přece nějakou barevnou paletu, přičemž ty nejdůležitější barvy se objevují hodně často v mnoha komponentách. Je dobré pro ně mít sémantické názvy, ne? Třeba u Rekol by `$c-main` byla ta růžová, protože tu barvu člověkv kódu potřebuje hodně. Souhlasím, že je mnohdy nesnadné ty barvy sémanticky pojmenovat, ale je to rozhodně lepší než hlavní odstíny hardcodovat (asi u půlky projektů, na kterých jsem kdy dělal, se v průběhu vývoje změnil design…) nebo je pojmenovávat stylem `$green`.